backend/src/Controller/AutocompleteController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class AutocompleteController extends AbstractController
+{
+    /**
+     * @Route("/ac/{place}", name="autocomplete", methods={"GET"})
+     */
+    public function index(string $place): JsonResponse
+    {
+        $google_url = "https://maps.googleapis.com/maps/api/place/autocomplete/json?input=$place&key=AIzaSyDZJTQ_zk-aXNr5gH8Si82_AOHiUmVXDJg&language=de";
```

Comment:
Be careful sharing your keys on GitHub. This should not be public and not be part of your source code.
Instead, you could make use of environment variables. Take a look at [`getenv`](https://www.php.net/manual/en/function.getenv.php) or if you like to have it more comfortable [`phpdotenv`](https://github.com/vlucas/phpdotenv). The latter is very similar to Node.js [`dotenv`](https://github.com/motdotla/dotenv#readme).

Reply:
Just as an addition because I was curious, you can find informations about environment variables in Symfony here:
https://symfony.com/doc/current/configuration.html#config-env-vars
And an even safer way with crypotgraphic keys here:
https://symfony.com/doc/current/configuration/secrets.html
Hope it helps, it doesn't look too complicated.

Reply:
Thank you both for your notes. I'll check it out. For me, the topic looks more complicated than it should be.

Reply:
Please take a look at your GitGuardian audit https://github.com/alex-jonas/capstone-project/pull/10/checks?check_run_id=1465070395
You seem to have even more secrets exposed. After hiding these it's even more important that you generate new secrets. These could still be accessed through your commit history even after removal

Reply:
Thank you, I'm aware of the issue. This is an old & invalid key. 

I am still trying to figure out how the .env file works with Symfony. Therefore I ask for patience :-)